app/src/main/java/org/openobservatory/ooniprobe/common/service/ServiceUtil.java

```diff
@@ -94,12 +95,13 @@ public static void callCheckInAPI(Application app) {
         testSuites.add(suite);
         testSuites.add(InstantMessagingSuite.initForAutoRun());
         testSuites.add(CircumventionSuite.initForAutoRun());
+        testSuites.add(PerformanceSuite.initForAutoRun());
```

Comment:
With respect to this, since we are looking at getting the same preferences for both foreground and background tests, the way to go about it was to make sure all enabled tests are available on the background test runs.

I agree some of the tests (`torsf`, `vanillator`) increase the overall runtime in the foreground.

Maybe we should have a more extensive conversation on this with @hellais , @agrabeli , @sloncocs